content/en/tracing/trace_collection/custom_instrumentation/go/otel.md

```diff
@@ -28,9 +28,12 @@ import (
 	"log"
 	"os"
 
-	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
-	ddotel "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/opentelemetry"
-	ddtracer "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext" // 1.x
+   // "github.com/DataDog/dd-trace-go/v2/ddtrace/ext" // 2.x
+	ddotel "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/opentelemetry" // 1.x
+   // "github.com/DataDog/dd-trace-go/v2/ddtrace/opentelemetry" // 2.x
+	ddtracer "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"  // 1.x
+   // "github.com/DataDog/dd-trace-go/v2/ddtrace/tracer" // 2.x
```

Comment:
```suggestion
	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext" // 1.x
	ddotel "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/opentelemetry" // 1.x
	ddtracer "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"  // 1.x
	// "github.com/DataDog/dd-trace-go/v2/ddtrace/ext" // 2.x
	// "github.com/DataDog/dd-trace-go/v2/ddtrace/opentelemetry" // 2.x
	// "github.com/DataDog/dd-trace-go/v2/ddtrace/tracer" // 2.x
```
It would be good to use spaces instead of tabs here